pkg/services/librarypanels/database.go

```diff
@@ -105,6 +154,33 @@ func (lps *LibraryPanelService) getAllLibraryPanels(c *models.ReqContext) ([]Lib
 	return libraryPanels, err
 }
 
+// getConnectedDashboards gets all dashboards connected to a Library Panel.
+func (lps *LibraryPanelService) getConnectedDashboards(c *models.ReqContext, uid string) ([]int64, error) {
+	connectedDashboardIds := make([]int64, 0)
```

Comment:
Nit, but I don't think there's any need to create the array, only to declare it.
```suggestion
	var connectedDashboardIds []int64
```

Reply:
Here I want the api to return an empty array instead of "null", so using `make` is correct in this case.

Reply:
A nil array should be treated the same as an empty one in Go. Can you see any practical difference?

Reply:
Yes, json.Marshal will treat a nil array as "null" value and I don't want that behavior here.

Reply:
Aha, I was wondering if there might be a difference there.